ambry-frontend/src/test/java/com.github.ambry.frontend/FrontendIntegrationTest.java

```diff
@@ -71,6 +74,9 @@
 public class FrontendIntegrationTest {
   private static final int SERVER_PORT = 1174;
   private static final ClusterMap CLUSTER_MAP;
+  private static final VerifiableProperties FRONTEND_VERIFIABLE_PROPS = buildFrontendVProps();
+  private static final int CHUNKED_GET_RESPONSE_THRESHOLD_BYTES =
```

Comment:
would recommend just making the config itself the static variable (in case other variables of it become useful down the line)